examples/visualization/plot_contour.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Hyper-parameter Relationship in Jupyter Notebook\n",
+    "# Visualizing Hyper-parameter Relationships\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
     "After optimizing the hyperparameter of neural networks, `plot_contour()` plots hyper-parameter relationship of completed trials in a study.\n",
```

Comment:
IMO, it is helpful to provide a link to the reference like [`plot_intermediate_values.ipynb`](https://github.com/pfnet/optuna/pull/667/files#diff-c93a73ade9fdafd6e829fe52ee95b2a6R10).
```suggestion
     "After optimizing the hyperparameter of neural networks, [`plot_contour()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_contour) plots hyper-parameter relationship of completed trials in a study.\n",
```